impeller/renderer/compute_unittests.cc

```diff
@@ -17,17 +17,10 @@
 namespace impeller {
 namespace testing {
 
-using ComputeTest = PlaygroundTest;
-INSTANTIATE_PLAYGROUND_SUITE(ComputeTest);
+using ComputeTest = ComputePlaygroundTest;
+INSTANTIATE_COMPUTE_SUITE(ComputeTest);
 
 TEST_P(ComputeTest, CanCreateComputePass) {
-  if (GetParam() == PlaygroundBackend::kOpenGLES) {
```

Comment:
Did you mean to remove these skips?

Reply:
Yes - this now gets skipped in the macro for setting up a compute playground. We can add Vulkan and GLES when/if we start supporting them.